view/console_view.py

```diff
@@ -79,6 +81,9 @@ def get_elevator_with_people(self) -> str:
         elevator_with_people = self._put_people_in_elevator(merged_people)
         return elevator_with_people
 
+    def get_stats(self) -> str:
+        return "<< Move: {}>>".format(self._bellhop_model.get_move_number())
```

Comment:
no tips??

Reply:
tips should be moved to and read from model

Reply:
yeah but do we actually want a secondary score apart from moves? the tips was mostly a joke. If we want to keep it I'll move to model

Reply:
I thought that efficiency etc were going to be in it too, only saying it score and putting it in not view. it's a good placeholder

Reply:
oh @thbenschop yeah no tips for same reason I said before. It's generated at completion by the view only. If we actually want it (?) throughout I will move the code and show there. 

I'm in favor of the only obj being to try to minimize moves for now but lmk